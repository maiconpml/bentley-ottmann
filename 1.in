3
3 1 3 7
2 2 5 2
2 1 5 7