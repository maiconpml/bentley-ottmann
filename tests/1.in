6
4 8 14 6
10 7 22 10
8 2 13 7
10.5 12 12 3
8 5 13 8
11 1 11 20