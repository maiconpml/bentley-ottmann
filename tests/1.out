12
4 8 14 6 8 5 13 8
8 2 13 7 11 1 11 20
4 8 14 6 11 1 11 20
8 5 13 8 11 1 11 20
10 7 22 10 11 1 11 20
10.5 12 12 3 11 1 11 20
10 7 22 10 10.5 12 12 3
8 5 13 8 10.5 12 12 3
4 8 14 6 10.5 12 12 3
8 2 13 7 10.5 12 12 3
8 5 13 8 10 7 22 10
4 8 14 6 8 2 13 7
